tests/integration/stub/teststubmain.cpp

```diff
@@ -455,10 +455,28 @@ int main() {
     Glib::init();
     Gio::init();
 
+    Glib::RefPtr<Glib::MainLoop> ml = Glib::MainLoop::create();
+
     TestImpl impl;
-    impl.connect(Gio::DBus::BUS_TYPE_SESSION,
-                     "org.gdbus.codegen.glibmm.Test");
+    guint connection_id = Gio::DBus::own_name(
+            Gio::DBus::BUS_TYPE_SESSION,
+            "org.gdbus.codegen.glibmm.Test",
+            [&](const Glib::RefPtr<Gio::DBus::Connection> &connection,
+                const Glib::ustring & /* name */) {
+                if (impl.register_object(connection, "/org/gdbus/codegen/glibmm/Test") == 0)
+                    ml->quit();
+            },
+            [&](const Glib::RefPtr<Gio::DBus::Connection> & /* connection */,
```

Comment:
You can use `Gio::DBus::SlotNameAppeared()` here

Reply:
Hum, I do not understand what you mean. Why would you need to watch for the name appearing in the service? Gio::DBus::own_name() takes care of owning the name, it already being taken, etc. etc..

Reply:
> creating the same service twice.

Gio::DBus::own_name() defines what happens in such a case (name_lost slot is called with a null connection).

Reply:
Ah, sorry. Read your comment again and understand what you mean now. Can use the type alias to instantiate an empty slot instead of specifying a lambda.

Will fix here and in README.md tomorrow.

Reply:
Done.